projects/plugins/jetpack/_inc/client/security/index.jsx

```diff
@@ -74,13 +74,22 @@ export class Security extends Component {
 		}
 
 		const foundProtect = this.props.isModuleFound( 'protect' ),
+			foundAccountProtection = this.props.isModuleFound( 'account-protection' ),
 			foundSso = this.props.isModuleFound( 'sso' ),
 			foundAkismet = this.isAkismetFound(),
-			rewindActive = 'active' === get( this.props.rewindStatus, [ 'state' ], false ),
+			rewindActive =
+				! isSearchTerm && 'active' === get( this.props.rewindStatus, [ 'state' ], false ),
 			foundBackups = this.props.isModuleFound( 'vaultpress' ) || rewindActive,
```

Comment:
I considered dropping the `rewindActive` check all along, given `isModuleFound()` already checks for the search term. But given this check is separate from the actual `vaultpress` module check, it makes me wonder if it's actually required for a different scenario.